http/src/main/resources/init-resources/jupyter-docker-compose-gce.yaml

```diff
@@ -31,6 +31,10 @@ services:
       PIP_USER: "false"
       PIP_TARGET: "${NOTEBOOKS_DIR}/packages"
       R_LIBS: "${NOTEBOOKS_DIR}/packages"
+      # The next two lines aren't great. But they're for updating PYTHONPATH, PATH in older than (inclusive) us.gcr.io/broad-dsp-gcr-public/terra-jupyter-base:1.0.2
+      # We should remove the two lines once we no longer support older images. In the meantime, we need to be careful updating Jupyter base images.
```

Comment:
Would be nice if we could link a ticket here. Seems like one of those things we'd forget to update even if we did no longer support the older images

Reply:
it's a bit unclear what the condition will be for the ticket. So for now, I'm adding a comment and added date when the comment is added